services/src/test/scala/cromwell/services/metadata/impl/MetadataDatabaseAccessSpec.scala

```diff
@@ -473,9 +473,29 @@ class MetadataDatabaseAccessSpec extends FlatSpec with Matchers with ScalaFuture
       } yield()).futureValue
     }
 
+    it should "properly query metadata summaries based on archived status and timestamp and update archive status after metadata deletion" taggedAs DbmsTest in {
+      (for {
+        workflowId1 <- succeededWorkflowMetadata(WorkflowId.fromString("11111111-1111-1111-1111-111111111112"))
+        workflowId2 <- succeededWorkflowMetadata(WorkflowId.fromString("11111111-1111-1111-1111-111111111113"))
+        _ <- dataAccess.refreshWorkflowMetadataSummaries(1000, Option(Long.MaxValue))
+        _ <- dataAccess.updateMetadataArchiveStatus(workflowId1, MetadataArchiveStatus.Archived)
+        responseEmpty <- dataAccess.queryRootWorkflowSummaryEntriesByArchiveStatusAndOlderThanTimestamp(Option("Archived"), OffsetDateTime.now().minusMinutes(1))
+        _ = responseEmpty shouldBe empty
+        _ = Thread.sleep(60000)
```

Comment:
Can we use an `eventually` check to return as soon as the condition is met (rather than adding another guaranteed 60s wait to this test run)?